docs/post-merge-analysis.md

```
@@ -0,0 +1,395 @@
+`apps/extractor` (commits `2355baa` squash, `8346f80` merge).
+
+Claim tags: **[confirmed]** = read the actual file/config. **[inferred]** =
+reasoned from naming/structure without direct confirmation.
+
+---
+
+## 1. Structure analysis
+
+**[confirmed]** Current top-level layout:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)